lib/foreman/logging.rb

```diff
@@ -0,0 +1,169 @@
+require 'logging'
+
+module Foreman
+  class LoggingImpl
```

Comment:
That is just a copy over from the original implementation in Katello. In the end, we could go either way as it's largely inconsequential given https://github.com/theforeman/foreman/pull/2376/files#diff-6bb962bdbc2f8a8c1ea2df38f09f8890R168 such that the object present in the environment is `Foreman::Logging`.